src/_data/site.json

```diff
@@ -8,8 +8,8 @@
         "d2fe": "https://boards.greenhouse.io/flowfuse/jobs/5185319004"
     },
     "messaging": {
-        "tagLine": "Elevate Node-RED with FlowFuse ",
-        "subtitle": "FlowFuse provides a complete platform for building, scaling and securing your production Node-RED applications. Accelerate digitalization of your industrial processes with ease.",
+        "tagLine": "The Fastest Way to Build and Operate Industrial Data Solutions",
+        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. We help manage, scale and secure your Node-RED solutions.",
         "title": "Node-RED for professionals",
```

Comment:
@Hasmin-AC @ZJvandeWeg, what could we replace this with to better align it with the new messaging?

Reply:
How about "Enterprise-ready Node-RED for Rapid Industrial Process Improvement"

Reply:
```suggestion
        "title": "Enterprise-ready Node-RED for Rapid Industrial Process Improvement",
```